bin/python_scripts/fix_organograms_s3_filenames.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Requirements: 
+# in ./bin/python_scripts directory >
+#   sudo /usr/lib/venv/bin/pip install -r requirements.txt
+#
+# Set up environment from ckan.ini
+# export AWS_ACCESS_KEY_ID=
+# export AWS_SECRET_ACCESS_KEY=
+# export AWS_REGION=
+# export AWS_ORG_BUCKET=
+# export S3_URL_PREFIX=
+# export POSTGRES_URL=<sqlalchemy.url from ckan.ini>
+#
+# Execute script like this -
+# python fix-organograms-s3-filenames.py dry-run
+#
+# after being run the SOLR search index needs to be refreshed:
+# paster --plugin=ckan search-index rebuild -r --config=/etc/ckan/ckan.ini
+
+import boto3
+import os
+import sys
+import psycopg2
+import subprocess
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+connection = psycopg2.connect(os.environ.get('POSTGRES_URL'))
+
+SKIP = COPIED = PURGED = True
+DONT_SKIP = NOT_COPIED = NOT_PURGED = False
+
+
+def setup_logging():
+    logger.setLevel(logging.INFO)
+    c_handler = logging.StreamHandler()
+    f_handler = logging.FileHandler('app.log')
+    _format = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')
+    c_handler.setFormatter(_format)
+    f_handler.setFormatter(_format)
+    logger.addHandler(c_handler)
+    logger.addHandler(f_handler)
+
+    logger.info('====================================================================')
+
+
+def get_path_parts(obj, s3_ls, i):
```

Comment:
Not a major thing and certainly not a blocker for this PR, but for this function you might find it easier to use [pathlib](https://docs.python.org/3/library/pathlib.html) rather than manually parsing the paths.

For example, you can do things like this:

```python
path = Path(obj.key)
dataset_name = path.parts[0]
filename = path_parts[-1]
directory = Path(*path.parts[:-1])

...

new_path = directory / new_filename
```

Reply:
will try it out